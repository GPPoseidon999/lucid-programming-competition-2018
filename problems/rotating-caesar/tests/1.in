Let's meet in the old treehouse at noon (12:00pm).