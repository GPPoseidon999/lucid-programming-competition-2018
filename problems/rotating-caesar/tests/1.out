Oiy'y tmnd tz gvt ecv mlzaemtsf cw rtuu (12:00xv).